output/Form.md

```diff
@@ -34,12 +49,18 @@ title: Yerun Open Science Award form
 
 -   *What are the steps you have taken/are taking to make the initiative
     available to others?*
+    
+Learnings from SYNCHROS and RESPOND. We have been developing SOPs (e.g., for anonymization), we are doing minutes in our meetings, we share our manuscripts in OneDrive so that we can keep track of versions, comments, etc.). We will share our server at the CCC with other research groups if they need it.
 
 -   *Is the practice, its structure and output, easily findable online? If yes,
     where?*
+    
+ This I don’t understand well. Include links to policy briefs, SOPs, SYNCHROS paper, Lea’s paper on Git, etc.
 
 -   *Can the practice be reused in other departments or institutions?*
 
+We will share our server at the CCC with other research groups if they need it. We will train other Departments. When we lead initiatives, we will develop an OS framework to work collaboratively (something from SYNCHROS here too). There are materials already available, e.g., DataShield monthly drop-in sessions.
+
 <!--# blank field, max 400 words -->
```

Comment:
Please note: There is only **ONE blank field** for this point, so all the content (addressing the three bullet points above) should go in a single place (400 words max.)